test/complex_include_test.rb

```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env rspec
 
 require_relative "./test_helper"
-require_relative "../../src/include/squid/complex"
+require_relative "../src/include/squid/complex"
```

Comment:
this really should not be needed. Proper Y2DIR env and Yast.include should just works.

Reply:
That's what I thought, but just removing it and using `Yast.include` was not enough. Due to the the way the test it's written other problems arose. I considered it to be kind of out of scope. Migrating the legacy tests is enough, I don't want to also spend time improving the pre-existing RSpec ones.